src/docs/get-started/flutter-for/android-devs.md

```diff
@@ -18,6 +18,7 @@ to use Flutter.
 This document can be used as a cookbook by jumping around and finding questions
 that are most relevant to your needs.
 
+{% include declarative.md %}
```

Comment:
To not distract the reader and let them dive straight into the meat of things, maybe we should add this into a small tip box with a link instead of inlining. We should add the box right after the "What is the equivalent of a View/UIView/Element".